overlord/ifacestate/ifacestate_test.go

```diff
@@ -176,10 +176,47 @@ type interfaceManagerSuite struct {
 	secBackend     *ifacetest.TestSecurityBackend
 	mockSnapCmd    *testutil.MockCmd
 	log            *bytes.Buffer
+	coreSnap       *snap.Info
```

Comment:
Are there any plugs or slots to remove from `repo_test.go`? I didn't check, just asking.

Reply:
No, I don't think so, they are still used by remaining tests.